colors/base16-3024.vim

```diff
@@ -150,8 +150,8 @@ let g:colors_name = "base16-3024"
 " Highlighting function
 " Optional variables are attributes and guisp
 function! g:Base16hi(group, guifg, guibg, ctermfg, ctermbg, ...)
-	let a:attr = get(a:, 1, "")
-	let a:guisp = get(a:, 2, "")
+	let l:attr = get(a:, 1, "")
+	let l:guisp = get(a:, 2, "")
```

Comment:
Maybe you can include https://github.com/chriskempson/base16-vim/pull/203 here, i.e. change this to use spaces.

Reply:
Good catch!